packages/release-trigger/src/release-trigger.ts

```diff
@@ -63,7 +63,7 @@ export const execFile = function (
   });
 };
 
-export const ALLOWED_ORGANIZATIONS = ['googleapis', 'GoogleCloudPlatform'];
+export const ALLOWED_ORGANIZATIONS = ['googleapis', 'GoogleCloudDataproc', 'GoogleCloudPlatform'];
```

Comment:
Looks like lint is failing.

Can you make the suggested change from below? (or run `npm run fix`)

Reply:
Done.